build.sh

```diff
@@ -643,36 +643,12 @@ case "$COMMAND" in
         ;;
 
     "verify-cocoapods")
-        pod setup
-        if [ -d .git ]; then
-          git diff-index --quiet HEAD || (
-            echo "Cannot test the podspec with uncommitted changes"
-            exit 1
-          )
-        fi
-        for podspec in Realm.podspec RealmSwift.podspec; do
-          sed -i '' "s|https://github.com/realm/realm-cocoa.git|$PWD|" $podspec
-          sed -i '' "s|:tag => \"v#{s.version}\"|:tag => \"v#{s.version}-pod-lint\"|" $podspec
-        done
-        if [ -d .git ]; then
-          git add *.podspec
-        else
-          git init
-          git add .
-        fi
-        git commit -m "commit for pod lint"
-        tag="v$(sh build.sh get-version)-pod-lint"
-        git tag $tag
-        pod spec lint --verbose || (
-          git tag -d $tag
-          git reset --hard HEAD^ || true
-          exit 1
-        )
-        git tag -d $tag
-        git reset --hard HEAD^ || true
         cd examples/installation
-        sh build.sh test-ios-objc-cocoapods || exit 1
-        sh build.sh test-ios-swift-cocoapods || exit 1
+        sh build.sh test-ios-objc-cocoapods
```

Comment:
The installation examples default to testing master if an environment variable named `sha` is not set. I'm not sure that's what we want.

The installation examples are also only capable of testing changes that have been pushed to GitHub. We should consider warning if there are unpushed changes on the branch we're testing so the user isn't surprised that changes aren't being tested when they should be.